Fabric.Identity.API/scripts/tests/CatalystDosIdentity.functional.tests.ps1

```diff
@@ -90,31 +90,365 @@ Describe 'Identity Cli Functional Tests' {
             }
         }
     }
-    
-    Describe 'Get-ClientRegistration' {}
-        
-    Describe 'New-ClientRegistration' {}
-    
-    Describe 'New-ImplicitClientRegistration' {}
-    
-    Describe 'New-HybridClientRegistration' {}
-    
-    Describe 'New-HybridPkceClientRegistration' {}
-    
-    Describe 'Invoke-UpdateClientRegistration' {}
-    
-    Describe 'Invoke-UpdateClientPassword' {}
-    
-    Describe 'Test-IsClientRegistered' {}
-    
+
+    Describe 'Client Flows' {
+        Context 'Happy Client Lifecycle' {
+            It 'Test, create, update, get, recreate, reset, test' {
+                $timeString = (Get-Date).ToString("hhmmss")
+                $testClientId = "CredentialsHappy$timeString"
+                $fabricToken = Get-FabricInstallerAccessToken -identityUrl $identityUrl -secret $installerSecret
+
+                # Test-IsClientRegistered (expect false)
+                $clientExists = Test-IsClientRegistered -identityUrl $identityUrl -clientId $testClientId -accessToken $fabricToken
+
+                $clientExists | Should -Be $false
+
+                # New-ClientCredentialsClientBody (expect object)
```

Comment:
what do these comments do?  are they important?

Reply:
they document the methods being called.   I will convert them to IT statements